src/metadata.rs

```diff
@@ -134,6 +134,8 @@ pub struct FighterSpawnMeta {
     #[serde(skip)]
     pub fighter_handle: Handle<FighterMeta>,
     pub location: Vec3,
+    // Set only for enemies.
+    pub trip_point_x: Option<f32>,
 }
```

Comment:
Do you think it makes sense just to have a default value here so we don't have to worry about unwrapping it later?

Threw a doc comment in there, too, just 'cause I had the time. :)

> **Note:** Suggestion is pseudo code and not tested.

```suggestion
    /// The point in the level players must pass for this enemy to start moving for the first time.
    ///
    /// This is only meaningful for enemy spawns, not for player spawns.
    ///
    /// A value of [`f32::MIN`] indicates that the enemy should start starts off moving by default.
    #[serde(default = default_f32_min)]
    pub trip_point_x: f32,
}

fn default_f32_min() -> f32 {
    f32::MIN
}
```

Reply:
It changes the semantics, however, on the other hands, it's a simplification without unwanted side effects, so I'll change it (requires also the unwrap() removal) :+1:

Reply:
Updated :smile: